playbooks/roles/handle-nodes-on-openstack/tasks/create_on_openstack.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Create volume
+  os_volume:
+    cloud: "{{ deploy_on_openstack_cloudname }}"
+    display_name: "{{ servername }}-vol"
+    size: 80
+    state: "present"
+
+- name: Create server
+  os_server:
```

Comment:
While changing this - what about switching to a heat stack? That would make naming, grouping and cleanup easier imo

Reply:
I love this idea, but I would prefer doing this in a later patchset.